src/Microsoft.DotNet.Build.Tasks/PackageFiles/versioning.targets

```diff
@@ -242,7 +241,6 @@
     <!-- ******* SourceLink Support, activated by setting $(UseSourceLink) ********* -->
 
     <!-- Generate the SourceLink file that will be passed to the C# compiler (using ProjectDir and GitHubRepositoryUrl as data) -->
-    <MakeDir Directories="$(BaseIntermediateOutputPath)" />
```

Comment:
You're going to still want this MakeDir for the RootIntermediateOutputPath.  There are official build scripts that directly call the `CreateOrUpdateCurrentVersionFile` target, and the directoy may not be created yet.